fe/fe-core/src/main/java/org/apache/doris/persist/EditLog.java

```diff
@@ -804,6 +805,21 @@ public static void loadJournal(Catalog catalog, JournalEntity journal) {
                     catalog.getAlterInstance().replayReplaceTable(log);
                     break;
                 }
+                case OperationType.OP_CREATE_SQL_BLOCK_RULE: {
```

Comment:
I think you missed the `readFields()` method of `JournalEntity.java`

Reply:
Yes, I forgot this when I migrated the code